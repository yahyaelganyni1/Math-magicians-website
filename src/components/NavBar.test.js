import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Navbar from './NavBar';

it('expect NavBar to match snapshot', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Navbar />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
