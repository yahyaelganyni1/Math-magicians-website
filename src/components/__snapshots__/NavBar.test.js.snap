// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect NavBar to match snapshot 1`] = `
<header
  className="nav-bar"
>
  <h1>
    Math Magicians
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</header>
`;
